src/dataflow/src/sink/kafka.rs

```diff
@@ -285,6 +289,21 @@ where
     };
 
     let name = format!("kafka-{}", id);
+
+    let encoder = Encoder::new(key_desc, value_desc, connector.consistency.is_some());
+    let key_schema_id = connector.key_schema_id;
+    let value_schema_id = connector.value_schema_id;
+    let stream = avro_encode_stream(
+        stream,
+        encoder,
+        key_schema_id,
+        value_schema_id,
+        connector.fuel,
+        name.clone(),
+    );
+
+    let stream = sort_on_one_worker(&stream, name.clone(), id);
+
```

Comment:
I know this wasn't explicitly part of your change but could we get a later commit that renames `s` on line 310 to something else `sink_state` maybe? as is `s` is hard to follow through the code